src/ess/reduce/time_of_flight/types.py

```diff
@@ -79,59 +68,25 @@ class FastestNeutron:
 This is typically of the order of 1-10 cm.
 """
 
-TimeOfArrivalResolution = NewType("TimeOfArrivalResolution", int | sc.Variable)
+TimeResolution = NewType("TimeResolution", int)
 """
-Resolution of the time of arrival axis in the lookup table.
-Can be an integer (number of bins) or a sc.Variable (bin width).
+Number of bins to use for the time of arrival (event_time_offset) axis in the lookup
+table. Should be around 1000.
```

Comment:
I presume this should be related to the instrument resolution? So instruments with a 1% resolution may need different values than ones with 5%?

Reply:
Do you think it's better to ask for a resolution in e.g. microseconds instead of a number of bins?

The reason why I did not do that was because the range needs to be exactly [0, 71ms], and if the 71ms is not a multiple of the bin width given by the user, we have to (silently?) modify the resolution that was actually given to us.
If they give a number of bins, we can guarantee that what they put in is reflected in the output.

That said, we could always go with the physical resolution if we properly document that we will guarantee at least what they asked for, but the resolution in the table could actually be a little finer?

Reply:
Update: I changed it to be a bin width with a unit, instead of an integer.